src/core/util/buildContext.js

```diff
@@ -1,5 +1,6 @@
 import Logger from "@reactioncommerce/logger";
 import ReactionError from "@reactioncommerce/reaction-error";
+import getSlug from "./getSlug.js"
```

Comment:
`getSlug` should not be added to the context. Import it as needed in the files that need it from the `api-utils` package: https://github.com/reactioncommerce/api-utils

Reply:
@mikemurray  done